lib/adroll/test.js

```diff
@@ -183,10 +183,12 @@ describe('AdRoll', function(){
           });
         });
 
-        it('should send events without revenue and order id', function(){
-          analytics.track('event', { revenue: 3.99 });
+        it('should send events with revenue and order id', function(){
```

Comment:
Nope, because I don't think that should have been there in the first place.
Definitely seems like no good reason to omit revenue on unmapped calls.
There really should be no distinction between mapped and unmapped except
that they can override the segment name on the event!

On Tue, Mar 10, 2015 at 6:02 PM, Prateek Srivastava <
notifications@github.com> wrote:

> In lib/adroll/test.js
> <https://github.com/segmentio/analytics.js-integrations/pull/557#discussion_r26181887>
> :
>
> > @@ -183,10 +183,12 @@ describe('AdRoll', function(){
> >            });
> >          });
> >
> > -        it('should send events without revenue and order id', function(){
> > -          analytics.track('event', { revenue: 3.99 });
> > +        it('should send events with revenue and order id', function(){
>
> should we leave in the old test for w/o revenue and order id?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/segmentio/analytics.js-integrations/pull/557/files#r26181887>
> .
>



-- 
Chris Sperandio
Success Engineer | Segment <http://segment.com/>